aiowebostv/webos_client.py

```diff
@@ -334,11 +344,7 @@ async def consumer_handler(
                     elif future is not None and not future.done():
                         self.futures[uid].set_result(msg)
 
-        except (
-            asyncio.CancelledError,
-            websockets.exceptions.ConnectionClosedError,
-            websockets.exceptions.ConnectionClosedOK,
-        ):
+        except asyncio.CancelledError:
```

Comment:
This looks like it will swallow cancellation
https://superfastpython.com/asyncio-task-cancellation-best-practices/

Reply:
Your comment makes sense and I have no idea why it is like that, I will try to remove it and see if there was a reason for it